NuGet.config

```diff
@@ -12,7 +12,9 @@
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
+    <add key="vssdk-archived" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk-archived/nuget/v3/index.json" />
```

Comment:
><add key="vssdk-archived" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk-archived/nuget/v3/index.json" /> [](http://example.com/codeflow?start=4&length=128)

Changes in this file look unrelated to the revert

Reply:
VS has changed the way they handle NuGet feeds since we last inserted this branch. This is a cherry pick from main that is required to build this branch.

Reply:
> This is a cherry pick from main that is required to build this branch.

The change shouldn't be part of this PR, unless it is specific to the change we are making. Which it sounds like it is not related.

Reply:
Ok, I can break it out into two PRs, but they'll be inserted in the same VS insertion either way. We can't insert without the feed change.